firestore/testapp/Assets/Firebase/Sample/Firestore/UIHandlerAutomated.cs

```diff
@@ -155,6 +155,8 @@ public class UIHandlerAutomated : UIHandler {
         TestInvalidArgumentAssertions,
         TestFirestoreDispose,
         TestFirestoreGetInstance,
+        TestWhereFilterQueries,
+        TestAndOrQueriesNoCompositeIndexes
```

Comment:
It looks like `TestAndOrQueriesNoCompositeIndexes` tests accidentally got deleted?

Reply:
Looks like I messed up resolving merge conflicts. It should be back now.